web/documentserver-example/java-spring/src/main/java/com/onlyoffice/integration/documentserver/models/filemodel/User.java

```diff
@@ -29,14 +31,17 @@
 @Getter
 @Setter
 public class User extends AbstractModel {
+    private FileStoragePathBuilder storagePathBuilder;
     private String id;
     private String name;
     private String group;
+    private String image;
 
     // the user configuration parameters
     public void configure(final int idParam, final String nameParam, final String groupParam) {
-        this.id = "uid-" + idParam;  // the user id
+        this.id = "" + idParam;  // the user id
```

Comment:
Have you tested other endpoints since these changes? I remember that we rely on this prefix in some places. Anyway, it doesn't appear to me that these changes are related to this request.